root/tree/tree/TTreeBenchmarks.cxx

```diff
@@ -1,3 +1,4 @@
+#include "../dataframe/RDataFrameBench.h"
```

Comment:
Is this some private header? Why do we need to include it that way

Reply:
Done!